spark/src/test/java/org/opensearch/sql/spark/asyncquery/AsyncQueryExecutorServiceImplSpecTest.java

```diff
@@ -313,6 +311,9 @@ public void interactiveQueryNoTimeout() {
     assertEquals(0L, (long) emrsClient.getJobRequest().executionTimeout());
   }
 
+  @Ignore(
```

Comment:
I got this yesterday and succedeed other times.
+1

Reply:
may related to key we used for testing, not our code issue.